src/main/java/com/example/bookstoreapp/dto/UpdateBookDto.java

```diff
@@ -0,0 +1,24 @@
+package com.example.bookstoreapp.dto;
+
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class UpdateBookDto {
+
+    @NotNull
```

Comment:
better use NotBlank for literals (but we will study how to validate the request body correctly)